### PR TITLE
test for ipaddr filter

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY
based upon work of mscherer

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
test/integration/targets/filters/files/requirements.txt
test/integration/targets/filters/tasks/main.yml

### CODE REVIEW

**Comment by Akasurde** [comment]
@mattclay how do I handle installation of `netaddr` on Python2 ?

File: test/integration/targets/filters/tasks/main.yml [hunk 1]
```diff
@@ -16,6 +16,11 @@
 # You should have received a copy of the GNU General Public License
 # along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
 
+- name: install required libraries
+  pip:
+    requirements: "{{ role_path }}/files/requirements.txt"
+  become: True
+
```
Comment (mattclay) [inline test/integration/targets/filters/tasks/main.yml:5]: Filter requirements must be installed before ansible-playbook is started, so you'll want to install them here:

https://github.com/ansible/ansible/blob/f49469f7bf42297ec47de16e7a5e851fbf1f442b/test/integration/targets/filters/runme.sh#L7

Comment (Akasurde) [reply test/integration/targets/filters/tasks/main.yml:5]: OK Cool

**Review by resmo — APPROVED** [review]
LGTM

**Comment by Akasurde** [comment]
@mscherer Thanks for the contribution. @resmo @mattclay Thanks for the review.

**Pull request merged**